allensdk/brain_observatory/behavior/data_objects/stimuli/util.py

```diff
@@ -63,3 +64,35 @@ def calculate_monitor_delay(sync_file: SyncFile,
         warnings.warn(warning_msg)
 
     return delay
+
+
+def get_image_stimulus_name(stimulus_file: BehaviorStimulusFile) -> str:
```

Comment:
please add this as a method on `BehaviorStimulusFile` as there similar methods there